doc/source/building/build_live_iso.rst

```diff
@@ -60,9 +61,9 @@ system from stick do the following:
 
       $ dd if=LimeJeOS-Leap-42.1.x86_64-1.42.1.iso of=/dev/<stickdevice>
 
-3. Run system from stick
+6. Start system from USB stick
 
-   Like in the example above use Qemu to run the system, but attach
+   Like in the example above use QEMU to start the system, but attach
```

Comment:
right, there is unfortunately no standard procedure, the only information which is true for everybody is that the boot from usb stick must be explicitly activated from the BIOS aka Firmware
which is before the operating System starts. May Firmware implementation offers a boot menu which can be activated at boot time via the F12 key, but as I said it's different in many ways